source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -281,6 +277,11 @@ described in the figure above.
    :start-after: begin rangeExample
    :end-before: end rangeExample
 
+The following figure shows the ``longIslandTriangle`` created.
```

Comment:
```suggestion
The following figure shows the polygon defined by the ``longIslandTriangle`` variable (triangle), and the locations of the movie theaters returned by our query (circles).
```
Suggestion: I think this description can be made more specific. 

Nit: As the figure shows the results, I think it could be moved after the results section.